lib/Zonemaster/Backend/DB/MySQL.pm

```diff
@@ -115,6 +115,68 @@ sub create_schema {
         );
     }
 
+    ####################################################################
+    # LOG LEVEL
+    ####################################################################
+    $dbh->do(
+        "CREATE TABLE IF NOT EXISTS log_level (
+            value INT,
+            level VARCHAR(15),
+
+            UNIQUE (value)
+        ) ENGINE=InnoDB
+        "
+    ) or die Zonemaster::Backend::Error::Internal->new( reason => "MySQL error, could not create 'log_level' table", data => $dbh->errstr() );
+
+    my ( $c ) = $dbh->selectrow_array( "SELECT count(*) FROM log_level" );
+    if ( $c == 0 ) {
+        $dbh->do(
+            "INSERT INTO log_level (value, level)
+            VALUES
+                (-2, 'DEBUG3'),
+                (-1, 'DEBUG2'),
+                ( 0, 'DEBUG'),
+                ( 1, 'INFO'),
+                ( 2, 'NOTICE'),
+                ( 3, 'WARNING'),
+                ( 4, 'ERROR'),
+                ( 5, 'CRITICAL')
+            "
+        );
+    }
+
+    ####################################################################
+    # RESULT ENTRIES
+    ####################################################################
+    $dbh->do(
+        "CREATE TABLE IF NOT EXISTS result_entries (
+            hash_id VARCHAR(16) NOT NULL,
```

Comment:
I wanted to avoid relying on the `id` field because auto incremental key is not supported in Clickhouse (a distributed database) and I wanted to keep the database internal as close as possible between each engine. Having said that, I realize it might be a little bit silly not to rely on specific features from each database engine.

> I'm sure that would be more efficient.

Maybe that won't be that bad since there is an index on `hash_id`. But to be honest I have no idea which one is better, I've not performed any test of this.

Reply:
> Maybe that won't be that bad since there is an index on hash_id.

Yeah, we could fix it later if it turns out it contributes to some bottleneck. Another thing that feels sub-optimal is that we use VARCHAR instead of CHAR for the hashes. But that probably doesn't make a huge difference either.